data/avro-schema/src/main/resources/avro/net/corda/data/membership/p2p/VerificationRequest.avsc

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "record",
+  "name": "VerificationRequest",
+  "namespace": "net.corda.data.membership.p2p",
+  "doc": "Request from MGM to verify our identity's membership data.",
+  "fields": [
+    {
+      "name": "destination",
+      "doc": "Holding identity of the member who needs to be verified.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "source",
+      "doc": "Holding identity of the MGM.",
+      "type": "net.corda.data.identity.HoldingIdentity"
+    },
+    {
+      "name": "requestId",
```

Comment:
I think it could be useful to put the ID of the registration into the request. The member could potentially use that to log the status of their registration request in the future.